analysers/analyser_osmosis_highway_deadend.py

```diff
@@ -442,8 +445,16 @@ def __init__(self, config, logger = None):
             resource = 'https://wiki.openstreetmap.org/wiki/Tag:service%3Ddrive-through')
 
         self.callback21 = lambda res: {"class": 1, "data": [self.way_full, self.node_full, self.positionAsText]}
-        self.callback22 = lambda res: {"class": 2, "data": [self.way_full, self.node_full, self.positionAsText]}
-
+        self.callback22 = lambda res: {
+            "class": 2,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": {"en": res[3]}
+        }
+        self.callback43 = lambda res: {
+            "class": 3,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": T_("Oneway inaccessible: {0}", res[3])
```

Comment:
Raw data for both, or text with translation for both.

Reply:
>or text with translation for both.

done